front/app/containers/EventsShowPage/components/InformationColumnDesktop.tsx

```diff
@@ -56,13 +56,7 @@ const InformationColumnDesktop = ({ event, className }: Props) => {
                     <EventAttendanceButton event={event} />
                   </Box>
                 )}
-                {event.attributes.attendees_count > 0 && (
-                  <ParticipantsCount
-                    count={event.attributes.attendees_count}
-                    maximumAttendees={event.attributes.maximum_attendees}
-                    isPastEvent={isPastEvent}
```

Comment:
- We can just pass `event` and extract the attributes inside `ParticipantsCount`. Reducing the "prop surface" in this manner has several benefits (which I won't elaborate on unless requested 😁).
- No need for `isPastEvent` with my suggested copy solution